main.go

```diff
@@ -31,8 +34,20 @@ var (
 
 func main() {
 	connpool := getConnectionPool()
+	defer connpool.Close()
+
+	kafkaWriter := kafka_helpers.GetWriter("ava-internship")
+	defer kafkaWriter.Close()
+
+	EsClient := elasticsearch_helpers.GetElasticsearchClient()
+	kafkaConsumer := kafka_helpers.NewConsumer("ava-internship", "localhost:9092", "group_id", EsClient, 500)
+	go kafkaConsumer.Consume()
+
+	retryConsumer := kafka_helpers.NewConsumer("retry", "localhost:9092", "group_id", EsClient, 15000)
```

Comment:
These params can be extracted to config. I suggest adding 1 config file with all params needed for Kafka, Elasticsearch, Postgresql and service and process it at the beginning when the service starts.